code/noRAM/app/src/test/java/com/example/noram/MilestoneUnitTest.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * This file is for testing the Milestone class.
+ */
+package com.example.noram;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.example.noram.model.Milestone;
+
+import org.junit.jupiter.api.Test;
+
+/**
+ * Tests for the Milestone class.
```

Comment:
author pwez

Reply:
added it there and on all the other test files